repos/spack_repo/builtin/packages/enzyme/package.py

```diff
@@ -41,23 +41,36 @@ class Enzyme(CMakePackage):
     depends_on("cxx", type="build")  # generated
     depends_on("fortran", type="build")  # generated
 
-    depends_on("llvm@7:12", when="@0.0.13:0.0.15")
-    depends_on("llvm@7:14", when="@0.0.32:0.0.47")
-    depends_on("llvm@7:14", when="@0.0.48:0.0.68")
-    depends_on("llvm@9:16", when="@0.0.69:0.0.79")
-    depends_on("llvm@11:16", when="@0.0.80:0.0.99")
-    depends_on("llvm@11:19", when="@0.0.100:0.0.148")
-    depends_on("llvm@15:19", when="@0.0.149:")
+    depends_on("libllvm@7:12", when="@0.0.13:0.0.15")
+    depends_on("libllvm@7:14", when="@0.0.32:0.0.47")
+    depends_on("libllvm@7:14", when="@0.0.48:0.0.68")
+    depends_on("libllvm@9:16", when="@0.0.69:0.0.79")
+    depends_on("libllvm@11:16", when="@0.0.80:0.0.99")
+    depends_on("libllvm@11:19", when="@0.0.100:0.0.148")
+    depends_on("libllvm@15:19", when="@0.0.149:")
     depends_on("cmake@3.13:", type="build")
```

Comment:
I think you might need a version of enzyme newer than the last spack version bump of Enzyme so it's worth bumping it here to add the latest release

Reply:
I was testing with `enzyme@main`. This version is automatically chosen with `mfem` since in the `mfem` Spack package, I'm requiring `enzyme@0.0.176:`, i.e. `enzyme` >= `0.0.176`.

In any case, I can add the latest Enzyme, v0.0.186 -- what are the supported LLVM versions for that?

Reply:
15 through latest are supported (and yeah the later versions have an update I worked with Julian on for amdgpu and mfem)

Reply:
Ok, I pushed an update with the latest `enzyme` version -- let me know if something does not look right.

Reply:
lgtm